tests/MongoDBDriverCursor_setTypeMap.phpt

```diff
@@ -6,20 +6,16 @@ $m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
 
 $c = new MongoDB\Driver\Command( [ 'drop' => 'test'] );
 try {
-	$cursor = $m->executeCommand( 'demo', $c );
-
-	foreach( $cursor->toArray() as $result )
-	{
-		var_dump($result);
-	}
+	$m->executeCommand( 'demo', $c );
 }
-catch ( InvalidArgumentException $e )
```

Comment:
I didn't see the need for catching an InvalidArgumentException here. If `executeCommand()` is called incorrectly, we can simply fail with an uncaught exception.